twake/backend/node/src/services/preview/src/previews/services/office.ts

```diff
@@ -0,0 +1,25 @@
+const unoconv = require("unoconv-promise");
+
+export async function convertFromOffice(
+  inputPath: string,
+  outputPath: string,
+  numberOfPages: number,
+) {
+  if (numberOfPages >= 1) {
+    const processOutputPath = outputPath.split("/");
+    processOutputPath.pop();
+    processOutputPath.push("tmp");
+    outputPath = `${processOutputPath.join("/")}/${outputPath.split("/").pop()}`;
```

Comment:
Avoid updating the input parameter, even if it is a string. You can create a new variable for this, name it `output`